src/meta/test/ProcessorTest.cpp

```diff
@@ -4,7 +4,6 @@
  *  (found in the LICENSE.Apache file in the root directory)
  */
 
-#include "base/Base.h"
```

Comment:
Why did you remove this?

Reply:
duplicate included, `base/Base.h` have imported by `src/meta/processors/BaseProcessor.h`